test/test_examples_2d.jl

```diff
@@ -105,6 +105,11 @@ const EXAMPLES_DIR = joinpath(pathof(Trixi) |> dirname |> dirname, "examples", "
             l2   = [0.022356422238096973],
             linf = [0.5043638249003257])
   end
+  @testset "parameters_mortar_alfven_wave.toml" begin
+    test_trixi_run(joinpath(EXAMPLES_DIR, "parameters_mortar_alfven_wave.toml"),
+            l2   = [6.457068474503248e-6, 2.2219242462067793e-6, 2.3062473272487904e-6, 2.2752088327881266e-6, 1.952277599926772e-6, 1.7865700364292728e-6, 1.8049416541526019e-6, 2.2647034913872624e-6, 1.1198821006370877e-6],
+            linf = [6.312646229511554e-5, 1.7349784189560347e-5, 1.6081522169253404e-5, 1.5587062945285335e-5, 9.627701115677567e-6, 1.1189762371577316e-5, 1.1315010600032593e-5, 1.9015998517543653e-5, 6.084398798398152e-6])
+  end
```

Comment:
I just ran this test on my machine and it took quite long (> 10s even after the first run). Maybe we can add `t_end=1.0` (or even lower) to the test to cut this in half? That would still mean more than 200 time steps in the test.

Reply:
I am okay with that. I can update the test